packages/flutter/lib/src/gestures/velocity_tracker.dart

```diff
@@ -225,6 +225,12 @@ class Velocity {
     return pixelsPerSecond == typedOther.pixelsPerSecond;
   }
 
+  Velocity operator -() => new Velocity(pixelsPerSecond: -pixelsPerSecond);
+  Velocity operator -(Velocity other) =>
+      new Velocity(pixelsPerSecond: pixelsPerSecond - other.pixelsPerSecond);
+  Velocity operator +(Velocity other) =>
+      new Velocity(pixelsPerSecond: pixelsPerSecond + other.pixelsPerSecond);
+
```

Comment:
also, please put these above the operator ==, so that == and hashCode are together.